boggle.js

```diff
@@ -1,9 +1,9 @@
 // let alphabet = ['A','B','C','D','E','F','G','H','I','J','K','L','M','N','O','P','Q','R','S','T','U','V','W','X','Y','Z']; alphabet[Math.floor(Math.random()*26)]
-let allArrays = ['AAEEGN', 'ELRTTY', 'AOOTTW',
+let dice = ['AAEEGN', 'ELRTTY', 'AOOTTW',
```

Comment:
👍

Reply:
Global constants are often written in ALL CAPS with underscores for spaces. This emphasizes the fact that they're meant to be a global constant, e.g.,

```javascript
let BOGGLE_DICE = [...];
```